### PR TITLE
Export ParseBool function from strconv package to template

### PR DESCRIPTION
**Pull request by n-rodriguez**
btw: the first lines of Go in my life :)

### CODE REVIEW

File: template.go [hunk 1]
```diff
@@ -346,6 +351,7 @@ func newTemplate(name string) *template.Template {
 		"keys":          keys,
 		"last":          arrayLast,
 		"replace":       strings.Replace,
+		"parseBool":     parseBool,
```
Comment (md5) [inline template.go]: You may just want to do `"parseBool": strconv.ParseBool` here since the function signature and return type isn't changing.

**Comment by md5** [comment]
Nice work!

**Comment by jwilder** [comment]
Thanks @n-rodriguez!

**Comment by n-rodriguez** [comment]
You're welcome! This is a very nice tool to have :)

**Pull request merged**